app/src/main/res/layout/item_fav_bus.xml

```diff
@@ -1,36 +1,59 @@
 <?xml version="1.0" encoding="utf-8"?>
-<androidx.cardview.widget.CardView xmlns:android="http://schemas.android.com/apk/res/android"
+<FrameLayout
```

Comment:
Is there a reason why this is a Frame instead of a CardView now?

Reply:
Using frame layout to display them on top of each other in a foreground/background style. The red delete color + trash icon is in a relative layout behind the card view.